inference-engine/thirdparty/movidius/XLink/pc/protocols/usb_boot.c

```diff
@@ -56,10 +56,11 @@ typedef struct {
 } deviceBootInfo_t;
 
 static deviceBootInfo_t supportedDevices[] = {
-    {
-        .pid = 0x2150,
-        .name = "ma2450"
-    },
+//    Myriad 2 device has been deprecated since 2020.4 release
+//    {
+//        .pid = 0x2150,
+//        .name = "ma2450"
+//    },
```

Comment:
`#if 0` looks better than multi-line comments. And more important - preserves git blame history of disabled content.